tests/cluster/cluster.tcl

```diff
@@ -20,7 +20,7 @@ proc get_cluster_nodes {id {status "*"}} {
         set node [dict create \
             id [lindex $args 0] \
             addr [lindex [split [lindex $args 1] ,] 0] \
-            shard-id [lindex [split [lindex [split [lindex $args 1] ,] 2] = ] 1]\
+            shard-id [lindex [split [lsearch -inline [split [lindex $args 1] ,] {shard-id=*}] =] 1]\
```

Comment:
We are not using this procedure to read nodename or hostname, here I made this change to make sure we read the shard-id correctly from CLUSTER NODES command output because in aux field now we have nodename and shard-id and nodename is optional parameter.